v0/destinations/woopra/data/WoopraTrackConfig.json

```diff
@@ -0,0 +1,99 @@
+[
+  {
```

Comment:
This also looks like same, as identify. Lets rename that file as common field mapping or user field mapping and call that same file for multiple functions. We do not need redundant files.